jenkinsfiles/RHEL_Check.groovy

```diff
@@ -95,7 +95,7 @@ pipeline {
 
     environment {
         def NUMBER = sh(script: "expr ${env.BUILD_NUMBER}", returnStdout: true).trim()
-        def TIMESTEMP = sh(script: "date +%s", returnStdout: true).trim()
+        def TIMESTAMP = sh(script: "date +%s", returnStdout: true).trim()
```

Comment:
TIMESTAMP is the correct spelling, and DATESTEMP should be changed to DATESTAMP to be consistent. @DoyleLi, is this change ok in all of the Jenkins files?

Reply:
Hi, Yes, it is OK to change the spelling everywhere.